ngraph_bridge/ngraph_cluster_manager.cc

```diff
@@ -32,13 +34,26 @@ size_t NGraphClusterManager::NewCluster() {
   return new_idx;
 }
 
-GraphDef* NGraphClusterManager::GetClusterGraph(size_t idx) {
+GraphDef* NGraphClusterManager::GetClusterGraphDef(size_t idx) {
   std::lock_guard<std::mutex> guard(s_cluster_graphs_mutex);
   return idx < s_cluster_graphs.size() ? s_cluster_graphs[idx] : nullptr;
 }
 
 void NGraphClusterManager::EvictAllClusters() { s_cluster_graphs.clear(); }
 
+Status NGraphClusterManager::GetClusterGraph(size_t idx, Graph* graph) {
+  GraphDef* cluster_gdef = NGraphClusterManager::GetClusterGraphDef(idx);
+  if (cluster_gdef == nullptr) {
+    graph = nullptr;  // is this ok?
+    return Status::OK();
```

Comment:
If we are returning Status, we should return an error status here i think

Reply:
I reverted some changes last friday, which are showing up again. I think while addressing review comments and I changed and pushed from a different repo which did not have the reverted changes. You should not be seeing this function. :)

Reply:
restored the PR. No changes to cluster manager